### PR TITLE
Add textbox view by default

### PR DESCRIPTION
**Pull request by c4d3r**
I think there should be a default View wrapper around textbox.
It creates more options, such as setting the bottom border only by default, changing background color easier, ... ([React Docs](http://facebook.github.io/react-native/releases/0.23/docs/textinput.html))

### CODE REVIEW

**Comment by gcanti** [comment]
@alvaromb what's your opinion? Is this backward compatible?

**Comment by alvaromb** [comment]
It will work with current `Form`s but it can change the style of your form because it is setting the `backgroundColor` of the container view to `white`. If it happens you have a custom bg color in your form it can look weird.

On the other hand, it is really handy to have a `View` wrapper to the `TextInput`s because it allows easier customizations without needing to create custom templates.

File: lib/stylesheets/bootstrap.js [hunk 1]
```diff
@@ -94,6 +94,14 @@ var stylesheet = Object.freeze({
       backgroundColor: DISABLED_BACKGROUND_COLOR
     }
   },
+  textboxContainer: {
+    normal: {
+      backgroundColor: 'white'
```
Comment (alvaromb) [inline lib/stylesheets/bootstrap.js]: Maybe we can leave the style empty to ensure backward compatibility.

File: lib/stylesheets/bootstrap.js [hunk 2]
```diff
@@ -94,6 +94,14 @@ var stylesheet = Object.freeze({
...
+    },
+    error: {
+      backgroundColor: 'rgba(255, 0, 0, 0.4)'
```
Comment (alvaromb) [inline lib/stylesheets/bootstrap.js]: Same as above.

File: lib/stylesheets/bootstrap.js [hunk 3]
```diff
@@ -178,4 +186,4 @@ var stylesheet = Object.freeze({
   }
 });
 
-module.exports = stylesheet;
+module.exports = stylesheet;
```
Comment (alvaromb) [inline lib/stylesheets/bootstrap.js]: No need to remove the new line to the end.

**Comment by gcanti** [comment]
Closing in favour of https://github.com/gcanti/tcomb-form-native/pull/282

**Pull request closed without merging**